Scripts/Core/EditorVR.Rays.cs

```diff
@@ -214,9 +214,17 @@ void OnProxyActiveChanged(IProxy proxy)
 				}
 			}
 
+			internal void UpdateRaycasts()
+			{
+				var intersectionModule = evr.GetModule<IntersectionModule>();
```

Comment:
If this happens every frame, then you might want to cache the intersection module.